addons/udes_stock/tests/test_stock_inventory.py

```diff
@@ -0,0 +1,89 @@
+from odoo.exceptions import ValidationError
+from . import common
+
+class TestStockInventory(common.BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        """Add a quant in a location to the setUpClass method"""
```

Comment:
Ok, I'm being super nit-picky and I don't like saying it, but some of your docstrings end with a full stop and some don't, I'm not going to hold anything up over that though!

Reply:
Changed to be consistent